### ISSUE TITLE
Implement proper ARIA functionality for Preferences page

### ISSUE THREAD

**Issue by ScrewTSW** [podman-desktop/podman-desktop#15163]
### Bug description

Currently Preferences page has a lot of functional elements with `role="none"` and missing `aria-label` components.

### Operating system

any

### Installation Method

Installer from website/GitHub releases

### Version

next (development version)

### Relevant log output

```shell

```

**Comment by odockal** [comment]
we should be ok with what we have right now, and this request requires more complex approach including whole new design to be done, in regards of aria functionality.

**Comment by danivilla9** [comment]
@ScrewTSW @odockal should we consider closing the issue? Otherwise I can try to add the aria-labels myself if you still consider this necessary
